eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/InMemoryLeaseManager.java

```diff
@@ -377,7 +379,7 @@ synchronized void removeLease(InMemoryLease goneLease) {
 
 
     private static class InMemoryLease extends CompleteLease {
-        private final static Logger TRACE_LOGGER = LoggerFactory.getLogger(InMemoryLease.class);
+        private static final Logger TRACE_LOGGER = LoggerFactory.getLogger(InMemoryLease.class);
```

Comment:
This is not right - we should not have static loggers in our SDK. Anyway, let's not fix this but I'm just pointing out that this is a code smell that shouldn't be accepted for future SDK development.